src/azure-cli/azure/cli/command_modules/rdbms/tests/latest/test_rdbms_commands.py

```diff
@@ -108,12 +108,28 @@ def _test_server_mgmt(self, database_engine, resource_group_1, resource_group_2)
         loc = 'eastus'
         default_public_network_access = 'Enabled'
         public_network_access = 'Disabled'
+        minimal_tls_version = 'TLS1_2'
+        default_minimal_tls_version = 'TLSEnforcementDisabled'
 
         geoGeoRedundantBackup = 'Disabled'
         geoBackupRetention = 20
         infrastructureEncryption = 'Enabled'
         geoloc = 'eastus'
 
+        list_checks = [JMESPathCheck('name', servers[0]),
```

Comment:
We had disabled a test related to restore due to test issue. I had added it back but it never made through could you merge my changes as well and regenerate the play back files ?

https://github.com/Azure/azure-cli/pull/12777

Reply:
added back